riseagain47/EPIC-Mail
# Ft-user-dashboard-#164276882

#### What does this PR do?
User should be able to create groups

#### Description of Task to be completed?
Have the following:
Create-group link/button
Text input fields for members of the groups
Add button for text input field
Text input field for group name
Create group button

#### How should this be manually tested?
GIVEN A user
WHEN User logs into their account dashboard
THEN User should see a create group link/button which links to a simple and descriptive form with text input fields, a corresponding add button and a create group button

#### Any background context you want to provide?
User should navigate to the create-group page by entering data into the login or signup form fields.

#### What are the relevant pivotal tracker stories?
#164276882 - User should be able to create groups
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/47803641/53625145-9c379400-3c02-11e9-9274-ac3976e89d96.png)
